cvat/apps/engine/migrations/0062_delete_previews.py

```diff
@@ -0,0 +1,43 @@
+import os
+import sys
+import traceback
+
+from django.db import migrations
+from django.conf import settings
+from cvat.apps.engine.log import get_migration_logger
+
+def delete_previews(apps, schema_editor):
+    migration_name = os.path.splitext(os.path.basename(__file__))[0]
+    with get_migration_logger(migration_name) as log:
+        def delete_object_previews(db_objects, root_path):
+            for db_obj in db_objects:
+                preview_path = os.path.join(root_path, str(db_obj.id), 'preview.jpeg')
+                try:
+                    os.remove(preview_path)
+                except Exception as e:
+                    log.error(f'Cannot delete path {preview_path}')
+                    log.error(str(e))
+                    traceback.print_exc(file=sys.stderr)
+
+        log.info('\nDeleting Data previews...')
+        Data = apps.get_model('engine', 'Data')
+        delete_object_previews(Data.objects.all(), settings.MEDIA_DATA_ROOT)
+
+        log.info('\nDeleting Job previews...')
+        Job = apps.get_model('engine', 'Job')
+        delete_object_previews(Job.objects.all(), settings.JOBS_ROOT)
```

Comment:
It can be a potentially a huge request.

Reply:
Results of my experiment:
```python
def f():
...   count = 0
...   now = datetime.now()
...   for ff in glob.glob('/home/django/data/jobs/*/preview.jpeg'):
...     count += 1
...   print(datetime.now() - now)
...   print(count)
...
>>> f()
0:03:49.090189
53651
```
vs
```python
def f():
... now =datetime.now()
... l = list(Job.objects.all())
... print(datetime.now() - now)
... print(len(l))
...
f()
0:00:09.785674
1015453
```
Both approaches require `os.remove(file_path)` for each iteration